docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,102 @@
+# VisualDL Web app Develop
+
+VisualDL has two three components.
+1. The Python/C++ SDK that logs the data during training.
+1. The single page client side app that visualized training data.
+1. The server(powered on Flask) that parses the saved data and delivers to the client side app for display.
```

Comment:
maybe say "parses the logs data", to be consistent with the first point.  Also, why are all the numbers "1."?

Reply:
The server (powered on Flask) that reads the logs data and delivers it to the client side app for displaying graphs (scalars/histograms) and embeddings.